### DISCUSSION TITLE
burner and radiative fraction

### DISCUSSION THREAD

**Discussion by 3dfirelab** [firemodels/fds#12844] [Q&A] [answered]
This is a question following this [discussion](https://groups.google.com/g/fds-smv/c/bstsk4oSTL4/m/-jAGnRLICgAJ) from the google group.
I am trying to understand where the radiative fraction set in REAC line apply.

for example in [methane_dx_1p25cm.fds](https://github.com/firemodels/fds/blob/master/Validation/UMD_Line_Burner/FDS_Input_Files/methane_dx_1p25cm.fds) form the UMD_Line_Burner you have 
```
&REAC RADIATIVE_FRACTION=0.0, N_SIMPLE_CHEMISTRY_REACTIONS=2, FUEL='METHANE', AIT_EXCLUSION_ZONE=-0.3,0.3,-0.05,0.05,-.05,0.15, AUTO_IGNITION_TEMPERATURE=600. / SFPE Table 2-8.2

&SURF ID='BURNER', SPEC_ID(1)='METHANE', MASS_FLUX(1)=0.04026, COLOR='RED'/ HRRPUA=2000 kJ/s/m2
```

if I set another burner linked to another SPEC_ID present in another REAC line, for example:
```
&REAC RADIATIVE_FRACTION=0.3, N_SIMPLE_CHEMISTRY_REACTIONS=2, FUEL='ACETONE', AIT_EXCLUSION_ZONE=-0.3,0.3,-0.05,0.05,-.05,0.15, AUTO_IGNITION_TEMPERATURE=600. / SFPE Table 2-8.2

&SURF ID='BURNER2', SPEC_ID(1)='ACETONE', MASS_FLUX(1)=0.04026, COLOR='RED'/ HRRPUA=2000 kJ/s/m2
```

when a flaming region is encouters (Xi_r q'''>10kW/m3), which radiative fraction is used?

**Comment by drjfloyd** [comment]
FDS first computes the source term using Eq 6.6.  FDS then volume integrates all the source term values in in cells with q''' > 10 (eq. 6.10).  It compares the volume integral to RADIATIVE_FRACTION times the heat release rate. If the volume integral is less, then FDS will multiply the source terms in those cells by whatever factor, C, is needed to scale the volume integral so it equals RADIATIVE_FRACTION time the heat release rate. If the volume integral is greater, then nothing is done.  So in the case with RADIATIVE_FRACTION=0, this means FDS only uses Eq 6.6 as any radiant emission from a grid cell with combustion will mean a radiative fraction greater than zero.

**Reply by rmcdermo** [reply, marked as answer]
Yes, but there is a subtle issue here in the example that @3dfirelab is providing.  In fire.f90 we have a scheme where the CHI_R(I,J,K) value is computed to be the HRR-weighted average of the CHI_R in a cell undergoing combustion.  In this example, though, the first reaction is not counted in that sum.  I think the intent of the user here is not being implemented in FDS.

I think that this notion of having FDS *predict* the radiative fraction (RADIATIVE_FRACTION=0) for one reaction mixed with specifying the radiative fraction for another should not be used by the user, and we probably should throw an error on this input.

**Reply by drjfloyd** [reply]
Thanks @rmcdermo. I misunderstood the question as replace the first two lines with the second and not in addition to. I agree that it doesn't make sense to combine prediction with specification, but if a user really wanted to do that for some reason, not sure we should prevent it.  Maybe a warning message?

**Reply by rmcdermo** [reply]
Sure.  I am fine with a warning.
